doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -616,7 +616,7 @@ nRF RPC libraries
 Other libraries
 ---------------
 
-|no_changes_yet_note|
+* Added a compression/decompression library with support for the LZMA decompression.
```

Comment:
Shouldn't this library be documented somewhere?

Reply:
Documentation will be added before the 2.8 release but after this PR